scripts/build_dist.sh

```diff
@@ -13,6 +13,7 @@ done
 ./scripts/link_page_resources.sh link
 
 cat dist/static/pipe/bootstrap.js | sed "s/^settings.bootstrapGristPrefix = .*/settings.bootstrapGristPrefix = XXX;/" > dist/latest.js
+cat dist/latest.js dist/static/pipe/components.js > dist/csv-viewer.js
```

Comment:
This is reasonable for now. Ideally, grist-static releases would be versioned, so users who prefer stability over freshness could choose that. We almost have that but it isn't documented, tested, and the versioned urls may be ugly. Still your PR isn't changing this, just adding another entrypoint. So fine to go with this for the demo.